cmd/list/logforwarders/cmd.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logforwarders
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "log-forwarders -c <cluster-id>"
+	short   = "List cluster log forwarders"
+	long    = "List log forwarders configured on a cluster, given a cluster ID"
+	example = "  # List all log forwarders on a cluster named \"mycluster\": " +
```

Comment:
Instead of using double quotes, this seems to be the standard way for defining examples:

  example = \`  # List all identity providers on a cluster named "mycluster"
    rosa list idps --cluster=mycluster\`